explorer/site-config/anvil-catalog/dev/index/workspaceEntityConfig.ts

```diff
@@ -29,44 +34,55 @@ export const workspaceEntity: EntityConfig<AnVILCatalogWorkspace> = {
         header: "Consortium",
         sort: {
           default: true,
-          sortKey: "consortium",
+          sortKey: ANVIL_CATALOG_FILTER_CATEGORY_KEYS.CONSORTIUM,
         },
         width: { max: "1fr", min: "120px" },
       },
       {
         componentConfig: {
           component: Components.Cell,
-          viewBuilder: ViewBuilder.buildDbGapId,
+          viewBuilder: ViewBuilder.buildTerraWorkspaceName,
         } as ComponentConfig<typeof Components.Cell>,
-        header: "dbGap Id",
+        header: "Terra Workspace",
         sort: {
-          default: true,
-          sortKey: "dbGapId",
+          sortKey: ANVIL_CATALOG_FILTER_CATEGORY_KEYS.WORKSPACE_NAME,
         },
-        width: { max: "1.24fr", min: "124px" },
+        width: { max: "1fr", min: "360px" },
+      },
+      {
+        componentConfig: {
+          component: Components.Links,
+          viewBuilder: ViewBuilder.buildStudyName,
+        } as ComponentConfig<typeof Components.Links>,
```

Comment:
Change component to `Link` see comments for explorer/site-config/anvil-catalog/dev/index/studiesEntityConfig.ts.